apps/air-discount-scheme/backend/src/app/modules/common/test/unit/auth.guard.spec.ts

```diff
@@ -7,7 +7,7 @@ const authGuard = new AuthGuard()
 
 describe('AuthGuard', () => {
   it('Api key should work for Ernir', async () => {
-    const request = {
+    const request: any = {
```

Comment:
I was unable to mock an express request and unable to cast this object to HttpRequest